Lombiq.Privacy.Samples/Recipes/Lombiq.Privacy.Samples.Content.recipe.json

```diff
@@ -0,0 +1,504 @@
+{
+  "name": "Lombiq.Privacy.Samples.Content",
+  "displayName": "Lombiq Privacy - Sample Content - Privacy, Forms, Workflows",
```

Comment:
I think it would be more manageable to split into 3 files:
- features and settings
- content and workflow definitions
- pages

Right now it's quite daunting.

Reply:
I think, I'm facing an OC issue again.

When I split the recipe into multiple files, it throws a `JavaScriptException: variables is not defined` if a `[js: variables('errorContentItemId')]` expression is present in the `InnerRecipe`...

Example to reproduce:

`recipe 1`:
```json
{
  "name": "Lombiq.Privacy.Samples",
  "displayName": "Lombiq Privacy - Sample - Privacy, Forms, Workflows",
  "description": "Sample for testing and demonstration of the Lombiq Privacy module.",
  "author": "Lombiq Technologies",
  "website": "https://github.com/Lombiq/Orchard-Privacy",
  "version": "1.0",
  "issetuprecipe": false,
  "categories": [
    "training"
  ],
  "tags": [
    "developer",
    "training",
    "sample"
  ],
  "variables": {
    "errorContentItemId": "erroritem00000000000000000",
    "registrationContentItemId": "registrationitem0000000000",
    "registrationWorkflowTypeId": "registrationworkflow000000",
    "registrationHttpRequestActivityId": "registrationhttpevent00000"
  },
  "steps": [
    {
      "name": "Recipes",
      "Values": [
        {
          "executionid": "Lombiq.Privacy.Samples",
          "name": "Lombiq.Privacy.Samples.FeaturesAndSettings"
        }
      ]
    }
  ]
}
```

`recipe 2`:
```json
{
  "name": "Lombiq.Privacy.Samples.Content",
  "displayName": "Lombiq Privacy - Sample - Privacy, Forms, Workflows",
  "description": "Sample for testing and demonstration of the Lombiq Privacy module.",
  "author": "Lombiq Technologies",
  "website": "https://github.com/Lombiq/Orchard-Privacy",
  "version": "1.0",
  "issetuprecipe": false,
  "categories": [
    "training"
  ],
  "tags": [
    "hidden"
  ],
  "steps": [
    {
      "name": "Content",
      "data": [
        {
          "ContentItemId": "[js: variables('errorContentItemId')]",
          "ContentItemVersionId": "[js: uuid()]",
          ...
        },
        ...
      ]
    }
  ]
}
```

If I copy the `Content` step from `recipe 2` into `recipe 1` everything works fine.

Did you faced something similar before?

Reply:
Yep, the `variables: {...}` object should be defined in `InnerRecipe` too, so they are not global

Reply:
That's unfortunate. Then keep it as one recipe.

Reply:
No, I can split the recipe into multiple files, the exception message was a bit confusing, the problem was not the script method `variables(...)` but the scope/visibility of the `"variables": {...}` object in the main recipe.
If I define the `"variables": {...}` inside `InnerRecipe` it works as expected.